tests/users/test_admin.py

```diff
@@ -314,3 +315,17 @@ def test_change_email(admin_client, caplog):
             emailaddress.verified,
         ),
     )
+
+
+@freeze_time("2025-06-16", tick=False)
```

Comment:
C’est la valeur par défaut.
```suggestion
@freeze_time("2025-06-16")
```